addons/functions/functions/fnc_vehicleAction.sqf

```diff
@@ -133,10 +133,10 @@ if (hasInterface) then {
 		true,
 		true,
 		"",
-		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1)", // _target, _this, _originalTarget
+		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1) && player == driver _target", // _target, _this, _originalTarget
 		50,
```

Comment:
reduce radius  to 10m or so (which is still very generous)

Reply:
```suggestion
		10,
```